packages/framework/client-logger/external-telemetry/src/common/consumers/index.ts

```diff
@@ -0,0 +1,15 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Base interface for all telemetry consumers.
```

Comment:
Since this is a core interface in the API, it would be good to offer a bit more detail in the docs here. Maybe even an `@example` block.

Reply:
Not opposed to this but wanted to clarify the direction. I was starting to think of the telemetry event interfaces in this package as "the contract" that we want to maintain, and the implementation, while also living in this package (for now, I mentioned to @seanimam that I could see it eventually being outside this package), it's just a reference implementation with no contract guarantees. In that case I'm not sure what the best tag would be for it... Maybe none? It'd still be visible when browsing the repo, which is where we probably want to have it as a reference implementation, but not exported.